lua/octo/reviews.lua

```diff
@@ -264,16 +265,16 @@ function M.add_review_comment(isSuggestion)
       {format("%d,%d", line1, line2), "OctoNvimDetailsValue"},
       {")"}
     }
-    writers.write_block({"", ""}, {bufnr = comment_bufnr, mark = false, line = 1})
-    api.nvim_buf_set_virtual_text(comment_bufnr, constants.OCTO_TITLE_VT_NS, 0, header_vt, {})
+    writers.write_block({"", ""}, {bufnr = comment_bufnr, line = 1})
+    writers.write_virtual_text(comment_bufnr, constants.OCTO_TITLE_VT_NS, 0, header_vt)
 
     if isSuggestion then
       local lines = api.nvim_buf_get_lines(props.content_bufnr, line1-1, line2, false)
-      writers.write_block({"```suggestion"}, {bufnr = comment_bufnr, mark = false})
-      writers.write_block(lines, {bufnr = comment_bufnr, mark = false})
-      writers.write_block({"```"}, {bufnr = comment_bufnr, mark = false})
+      writers.write_block({"```suggestion"}, {bufnr = comment_bufnr })
+      writers.write_block(lines, {bufnr = comment_bufnr })
+      writers.write_block({"```"}, {bufnr = comment_bufnr })
     else
-      writers.write_block({""}, {bufnr = comment_bufnr, mark = false})
+      writers.write_block({""}, {bufnr = comment_bufnr })
     end
```

Comment:
Now I selected multiple lines with modified code. Just too see how that looks later in the PR buffer.
I had some issues to figure our which lines I actually could introduce in the selection for a comment. Do you know the technical details. Then we might could think of a solution approach for this.
But first let's see how this looks.

PS: Because this virtual text in the comment buffer tells me its the `[LEFT]` version I comment. For some reason I now get a horizontal split for the diff an nomore a vertical one... :thinking:

Reply:
Hmm. Everything is red. Why? And there is this little box-line-issue on the right. 

![octo_multiline](https://user-images.githubusercontent.com/12543647/109435760-59d90580-7a1c-11eb-9229-079e0bb809db.png)

Reply:
> I had some issues to figure our which lines I actually could introduce in the selection for a comment. Do you know the technical details

You should visual select the lines and then `<space>ca` or `Octo comment add`

Reply:
> PS: Because this virtual text in the comment buffer tells me its the [LEFT] version I comment. For some reason I now get a horizontal split for the diff an nomore a vertical one...

That should not be the case, can you reproduce it and send me an screenshot?

Reply:
> Hmm. Everything is red. Why? 

Comments in left side is highlighted with DiffDelete. I think GH UI does something similar (use green for right side and red for the left one) but will double check

> And there is this little box-line-issue on the right.

Argh, I though I got rid of that one. It happens on empty lines, but I thought I fixed it 🙃

Reply:
> You should visual select the lines and then <space>ca or Octo comment add

Yes. That's what I did. And then I got twice the error that my selection is too big and I can only select parts with changes hunks (phrasing this roughly from my memories). I reduced my selection until it was successful.

Reply:
ah, ok. Yep GitHub only allows you to place comments on lines that appear in the diff hunk. You are not allowed to comment on any arbitrary line apparently

Reply:
My tons of comments have become way too much. I'll open issues where we can discuss such minor problems like this one. They are in fact unrelated to this PR itself. They are just caused by me using the plugin more intensively for testing the PR. I'll close the conversation as soon as I wrote the issue.

Reply:
#108 is the issue

I can't close the conversation even I started it. @pwntester could you help to do this. So we could start to get rid of my chaos I started here... :see_no_evil: